docs/en/api/POSIX-API.md

```diff
@@ -370,7 +387,7 @@ For example, the data cached on Node A might be stale if the file is deleted and
 
 FUSE has the following I/O modes controlling whether data will be cached and the cache invalidation policy:
 - `direct_io` (default): disables the kernel data cache
-- `kernel_cache`: always cache data in kernel. This should only be enabled on filesystem, whether the file data is never changed externally (not through the mounted FUSE filesystem)
+- `kernel_cache`: always cache data in kernel and no cache invalidation is happening. This should only be enabled on filesystem, whether the file data is never changed externally (not through the current FUSE mount point)
```

Comment:
I guess you are trying to say `"where" the file data`, which was why I couldn't understand

Reply:
This option disables flushing the cache of the file contents on every open(2). This should only be enabled on filesystems where the file data is never changed externally (not through the mounted FUSE filesystem). Thus it is not suitable for network filesystems and other intermediate filesystems.


yeah.... should be "where" i didn't copy correctly